src/PlexShareScreenshare/Server/ScreenStitcher.cs

```diff
@@ -0,0 +1,124 @@
+﻿
+///<author>Aditya Agarwal</author>
+///<summary> 
+///This file contains the ScreenStitcher Class that implements the
+///screen stitching functionality. It is used by ScreenshareServer. 
+///</summary>
+
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Windows.Controls;
+using static System.Net.Mime.MediaTypeNames;
+
+
+// Each frame consists of the resolution of the image and the ImageDiffList
+using Frame = System.Tuple<System.Tuple<int, int>,
+                        System.Collections.Generic.List<System.Tuple<System.Tuple<int, int>,
+                        System.Tuple<int, int, int>>>>;
+
+
+namespace ScreenShare.Server
+{
+    /// <summary>
+    /// Class contains implementation of the screen stitching using threads (tasks)
+    /// </summary>
+    internal class ScreenStitcher
+    {
+        private SharedClientScreen _sharedClientScreen;
+
+        private Task _stitchTask;
+        private Bitmap _stichedImage;
+
+        // new frame to hold the updated pixels
+        private Frame _newFrame;
+        private Bitmap _oldImage;
+        private Tuple<int, int> _resolution;
+
+        // Token and its source for killing the task
+        private CancellationTokenSource Source;
+        private CancellationToken Token;
+
+        // Called by the `SharedClientScreen`
+        ScreenStitcher(SharedClientScreen scs)
```

Comment:
Constructor should be public

Reply:
scs variable is never used. Why do we need it here? Are you sure this is how the ScreenStitcher is going to get constructed?